datasets/ONS-Trade-in-goods/imports.py

```diff
@@ -32,68 +90,89 @@
                              }, na_values=['','N/A'], keep_default_na=False)
 data
 
+
 # %%
+
+# In[10]:
+
+
+pd.set_option('display.float_format', lambda x: '%.0f' % x)
+
+
+# In[11]:
+
+
 table = data.drop(columns='DIRECTION')
 table.rename(columns={
     'COMMODITY': 'CORD SITC',
     'COUNTRY': 'ONS Partner Geography'}, inplace=True)
 table = pd.melt(table, id_vars=['CORD SITC','ONS Partner Geography'], var_name='Period', value_name='Value')
-table['Period'] = table['Period'].astype('category')
+#table['Period'] = table['Period'].astype('category')
```

Comment:
can go

Reply:
Thank you so much for the PR review and feedback. I will tidy up the code shortly and merge to master.